maint.php

```diff
@@ -214,13 +215,13 @@ function form_actions() {
 				if (get_request_var('drp_action') == '1') { /* associate */
 					for ($i=0;($i<count($selected_items));$i++) {
 						db_execute_prepared('REPLACE INTO plugin_maint_hosts (type, host, schedule)
-							VALUES (1, ?, ?)',
+							VALUES (' . HOST_TYPE_HOSTS . ', ?, ?)',
 							array($selected_items[$i], get_request_var('id')));
```

Comment:
Defining the constant is a good way to go, but this should still be the first parameter within the array for consistency and continuity. Also, it would probably be wise to name the constant MAINT_HOST_TYPE_HOSTS to avoid potential conflicts with the core, or put a `if !defined` around the constant definition after all includes are performed.

Reply:
The constants have been renamed.  The constants have been moved to SQL parameters.